pkg/dump/dump.go

```diff
@@ -59,6 +65,68 @@ type ChunkMeta struct {
 	RowsLen int
 }
 
+type Writers struct {
+	gzw *gzip.Writer
+	tw  *tar.Writer
+	ew  *cipher.StreamWriter
+}
+
+type Readers struct {
+	gzr *gzip.Reader
+	tr  *tar.Reader
+	er  *cipher.StreamReader
+}
```

Comment:
Don't use `Writers` and `Readers` names for these structs. These names describe the contents of these structures.

Let's call it `Writer` and `Reader` to describe what they do. Also, it would be great if these structures could implement `io.Writer` and `io.Reader` interfaces.